### PR TITLE
More detailed issue templates

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/--bug-report.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+This is an issue tracker, used to manage and track the development of **Alpakka Kafka**. 
+
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka:      https://github.com/akka/akka/issues 
+ - Akka HTTP: https://github.com/akka/akka-http/issues 
+ - Alpakka:   https://github.com/akka/alpakka/issues 
+
+This is not a support system, ask questions or discuss ideas in the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/streams-and-alpakka).
```
Comment (Philippus) [inline .github/ISSUE_TEMPLATE/--bug-report.md]: `please` or `you can` makes it sound less commanding.

File: .github/ISSUE_TEMPLATE/--bug-report.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+about: Create a report to improve Alpakka Kafka
+---
+
+### Are you looking for help?
+
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+This is an issue tracker, used to manage and track the development of **Alpakka Kafka**. 
+
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
```
Comment (2m) [inline .github/ISSUE_TEMPLATE/--bug-report.md]: Maybe s/specific/other

File: .github/ISSUE_TEMPLATE/--bug-report.md (hunk 1 above)
Comment (2m) [reply .github/ISSUE_TEMPLATE/--bug-report.md]: Yes. The first part could go away entirely.

> Please ask questions or discuss ...

File: .github/ISSUE_TEMPLATE/--bug-report.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ - Akka:      https://github.com/akka/akka/issues 
+ - Akka HTTP: https://github.com/akka/akka-http/issues 
+ - Alpakka:   https://github.com/akka/alpakka/issues 
+
+This is not a support system, ask questions or discuss ideas in the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/streams-and-alpakka).
+
+We've collected a few tips for [debugging Alpakka Kafka](https://doc.akka.io/docs/akka-stream-kafka/current/debugging.html).
```
Comment (2m) [inline .github/ISSUE_TEMPLATE/--bug-report.md]: Could lead with an expected results for debugging, for example

> To get debug level logging and for other troubleshooting tips, take a look at ...

File: .github/ISSUE_TEMPLATE/--question.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+name: "❓ Question"
+about: Please use https://discuss.lightbend.com/c/akka/streams-and-alpakka for questions
+
+---
+
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+Otherwise, please use the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/streams-and-alpakka) for questions instead of posting them to the issue tracker.
```
Comment (marcospereira) [inline .github/ISSUE_TEMPLATE/--question.md]: Should this also _warns_ users that the issue will probably be closed without further action?

**Pull request merged**